feat(to-camel-case): init toCamelCase

Mercyfulsin opened the pull request:
My attempt at the toCamelCase feature.
Fairly new to PR's on open source so most of my files were created by referencing other projects like toLower or toSucc.
yarn test passed but failed for plexis > index.js, unsure if this is related to my files? Seems to have been issues with toPred.

Fixes #14

vorillaz commented on packages/plexis/package.json:
```diff
@@ -13,6 +13,7 @@
     "@plexis/to-pred": "^0.0.12",
     "@plexis/to-succ": "^0.0.12",
     "@plexis/to-title": "^0.0.12",
+    "@plexis/to-camel-case": "^0.0.12",
```
Can you please move this one to alphabetic order?

vorillaz commented on packages/toCamelCase/package.json:
```diff
@@ -0,0 +1,34 @@
+  ],
+  "author": "Andres Ruiz-Velediaz <aruiz858@gmail.com>",
+  "homepage": "https://github.com/Mercyfulsin/plexis#readme",
+  "license": "ISC",
```
Hey, the license is wrong as well as some keys for the package. 
You can create the new package using the template from here https://github.com/plexis-js/plexis/tree/master/.package-template

vorillaz commented on packages/plexis/src/index.js:
```diff
@@ -1,6 +1,7 @@
 export {default as toPred, default as toPredecessor} from '@plexis/to-pred';
 export {default as toSucc, default as toSuccessor} from '@plexis/to-succ';
 export {default as toTitle, default as titleize} from '@plexis/to-title';
+export {default as toCamel, default as toCamelCase} from '@plexis/to-camel-case';
```
According to the issue you shall export 3 methods `toCamelCase, camelize, camelCase` #14

vorillaz commented on packages/toCamelCase/README.md:
```diff
@@ -0,0 +1,25 @@
+# `@plexis/toCamelCase`
+
+Convert given string to camel case
```
There is a tiny typo here.

vorillaz commented:
Hey, @Mercyfulsin no worries at all. First things first thanks for your efforts. Each time you are adding a new method into the main `plexis` package the test is validating that everything is exported as expected. There is a snapshot testing against the main package.
You can use `yarn test -u` in order to update the snapshot.
More infos can be found here https://jestjs.io/docs/en/snapshot-testing
Also, since I have merged some packages you shall rebase and solve any conflicts.

Mercyfulsin commented:
I keep getting caught with this error when attempting to update the snapshots.
Any idea? I updated the aliases and changed my package license.
![Screenshot_1](https://user-images.githubusercontent.com/16928058/67346320-2bc3e700-f4f3-11e9-89cc-2687fc624e27.png)

Mercyfulsin commented:
Ah I just saw the yarn bootstrap tip. I did that and attempted it again and got the same error but at a different point in the code?
![Screenshot_1](https://user-images.githubusercontent.com/16928058/67346511-d76d3700-f4f3-11e9-8e62-05997a984acf.png)
